scripts/qa_crowbarsetup.sh

```diff
@@ -90,6 +90,16 @@ case "$cloud" in
         vlan_fixed=589
         want_ipmi=true
     ;;
+    qa1)
+        nodenumber=6
+        net=${netp}.26
+        net_public=$net
+        vlan_public=300
+        #vlan_admin=610
```

Comment:
admin vlan is wrong (even if the value is not used in the code, it can confuse people)